app/models/wpcc/salary_frequency.rb

```diff
@@ -15,7 +15,8 @@ def initialize(attributes)
     end
 
     def to_adj
-      @converter.adjectives[locale.to_s][to_s]
+      frequency = to_s
```

Comment:
`frequency = to_s` what's been converted to string here. Could you make it more _obvious_ first?